### PR TITLE
Add the information about /tmp

### PR DESCRIPTION
**Pull request by balhar-jakub**
Describe your pull request here:
Add the notes about the standard behavior of /tmp

List the file(s) included in this PR:
docs/user-guide/installandconfig.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/installandconfig.md [hunk 1]
```diff
@@ -17,6 +17,18 @@ zFS is a UNIX file system where Zowe runtime files and folders are installed. Zo
 
 For more information about USS, see [Addressing UNIX System Servies (USS) Requirements](./configure-uss).
 
+:::note
+Zowe runs in USS, it makes use of shell scripts and TCP/IP sockets. The amount of open I/O handles and the sizes that you may see are the result of normal use of the above resources. If you do not observe any other malfunction, the number of open files should not be a concern.
```
Comment (janan07) [inline docs/user-guide/installandconfig.md:21]: The sizes of what? Where would malfunction be detected? What type of malfunction...insufficient available memory?

File: docs/user-guide/installandconfig.md [hunk 2]
```diff
@@ -17,6 +17,18 @@ zFS is a UNIX file system where Zowe runtime files and folders are installed. Zo
...
+
+If you are concerned about the above behavior please update the following property to move the opened files to different directory:
```
[the remark this answers is not in the document]
Comment (balhar-jakub) [reply docs/user-guide/installandconfig.md:23]: Yes.

File: docs/user-guide/installandconfig.md (hunk 1 above)
Comment (balhar-jakub) [reply docs/user-guide/installandconfig.md]: ... like reaching the socket limits or limit on the amount of files in the specific directory ...

**Review by janan07 — APPROVED** [review]
reviewed and edited by doc squad

**Pull request merged**